inference-engine/samples/speech_sample/README.md

```diff
@@ -109,6 +109,8 @@ Options:
                             If you use the cw_l or cw_r flag, then batch size and nthreads arguments are ignored.
     -cw_r "<integer>"       Optional. Number of frames for right context windows (default is 0). Works only with context window networks.
                             If you use the cw_r or cw_l flag, then batch size and nthreads arguments are ignored.
+    -oname "<outputs>"      Optional. Change the order of output layers.
```

Comment:
It's unclear. Why not add here full text from speech sample help, similarly to other items?